mountpoint-s3/src/inode.rs

```diff
@@ -603,6 +662,10 @@ impl WriteHandle {
                 error!(inode=?self.ino, "inode already exists");
                 Err(InodeError::InodeNotWritable(self.ino))
             }
+            WriteStatus::Deleted => {
+                error!(inode=?self.ino, "inode is deleted");
+                Err(InodeError::InodeDeleted(self.ino))
```

Comment:
Same thing here about `InodeDeleted` being the same thing as `InodeDoesNotExist`?